src/libcommonserver/io/iohelper.cpp

```diff
@@ -591,6 +591,15 @@ bool IoHelper::checkIfPathExistsWithSameNodeId(const SyncPath &path, const NodeI
             LOGW_WARN(logger(), L"Error in IoHelper::getNodeId for path=" << Path2WStr(path).c_str());
```

Comment:
Out of scope, but nice to have:
```suggestion
            LOGW_WARN(logger(), L"Error in IoHelper::getNodeId for " << Utility::formatSyncPath(path).c_str());
```

Reply:
Done